Stormshield/stormshield_module/base.py

```diff
@@ -0,0 +1,97 @@
+from functools import cached_property
+from posixpath import join as urljoin
+from typing import Any
+import re
+import requests
+from requests import RequestException, Response
+from tenacity import (
+    RetryError,
+    Retrying,
+    retry_if_exception_type,
+    stop_after_attempt,
+)
+
+from sekoia_automation.action import GenericAPIAction
+
+
+class StormshieldAction(GenericAPIAction):
+    endpoint: str
+
+    @cached_property
+    def api_token(self):  # type: ignore
+        return self.module.configuration.get("api_token")
```

Comment:
The `api_token` is required. The connector must fail if not defined.
```suggestion
        return self.module.configuration["api_token"]
```